packages/fluent/lib/swatches/fluent-main-dark.json

```diff
@@ -6,11 +6,11 @@
     "version": "5.1.2-dev.7",
     "api": "modern",
     "previewColors": [
-        "#ffffff",
-        "#f5f5f5",
-        "#ff6358",
-        "#d6534a",
-        "#424242"
+        "#1b1a19",
+        "#21201f",
+        "#2899f5",
+        "#6cb8f6",
```

Comment:
The color here should be secondary -> https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/lib/swatches/fluent-main-dark.json#L183